tests/utils_for_tests.py

```diff
@@ -8,12 +8,48 @@
 
 import logging
 from importlib import import_module
+import numpy as np
 
 from elfpy.simulators import Simulator
 from elfpy.utils import sim_utils
 import elfpy.utils.parse_config as config_utils
 
 
+def float_to_string(value, precision=3, min_digits=0, debug=False):
```

Comment:
If you think this function would be useful as a potential output utility, then I'd suggest putting it in the output utils file. I don't really see a reason to put it here, especially considering that it is not being called. Also, keep in mind that if you delete it and later find a use for it you can always look in the git history to find it and revive it.

Reply:
i put this in utils_for_tests for now, which also lives outside of the package. is that sufficient for now?